packages/@dcl/sdk-commands/src/logic/scene-validations.ts

```diff
@@ -14,6 +15,17 @@ export function getSceneFilePath(projectRoot: string): string {
   return resolve(projectRoot, SCENE_FILE)
 }
 
+/**
+ * Returns `scene.json` file contents as JSON.
+ * @param components The necessary components to read the file
+ * @param projectRoot The path to the directory containing the scene file.
+ */
+export async function getSceneJSON(components: Pick<CliComponents, 'fs'>, projectRoot: string) {
+  const sceneJsonRaw = await components.fs.readFile(getSceneFilePath(projectRoot), 'utf8')
+  const sceneJson = JSON.parse(sceneJsonRaw) as Scene
+  return sceneJson
+}
```

Comment:
exporting this function is unnecessary, use or rename `validateSceneJson` instead